File: app/src/test/java/com/yatatsu/passiveviewsample/dagger/module/MockNetworkModule.java
package com.yatatsu.passiveviewsample.dagger.module;


import android.content.Context;

import com.squareup.okhttp.OkHttpClient;

public class MockNetworkModule extends NetworkModule {

    @Override
    public OkHttpClient provideOkHttpClient(Context context) {
        return new OkHttpClient();
    }
}

File: app/src/test/java/com/yatatsu/passiveviewsample/dagger/module/MockNetworkModule.kt
package com.yatatsu.passiveviewsample.dagger.module


import android.content.Context

import com.squareup.okhttp.OkHttpClient

class MockNetworkModule : NetworkModule() {

  override fun provideOkHttpClient(context: Context): OkHttpClient {
    return OkHttpClient()
  }
}